lib/kafo/configuration.rb

```diff
@@ -90,6 +91,14 @@ def set_custom(key, value)
       custom_storage[key.to_sym] = value
     end
 
+    def get_custom_fact(key)
```

Comment:
in ruby we typically do this

```
def custom_fact(key)
  custom_fact_storage[key.to_s]
end

def custom_fact=(key, value)
  custom_fact_storage[key.to_s] = value
end
```

but given method names above whose author probably knew what he was doing (:man_facepalming:) I'm happy to keep as is, just wanted to let you know in case you haven't seen this before

Reply:
I've seen it but decided to copy what was already there for a consistent API.